sympy/core/expr.py

```diff
@@ -2849,12 +2848,10 @@ def series(self, x=None, x0=0, n=6, dir="+", logx=None):
             raise ValueError("Dir must be '+' or '-'")
 
         if x0 in [S.Infinity, S.NegativeInfinity]:
-            sgn = 1 if x0 is S.Infinity else -1
-            s = self.subs(x, sgn/x).series(x, n=n, dir='+')
-            if n is None:
-                return (si.subs(x, sgn/x) for si in s)
-            return s.subs(x, sgn/x)
-
+            s = self.aseries(x, n)
+            if x0 is S.NegativeInfinity:
+                return s.subs(x, -x)
+            return s
```

Comment:
I am not sure about this. Let's not change the default for now. We can integrate this later. We need to be sure ``aseries`` will cover all the cases which ``series`` covers also maintaining correctness.